docs/automotive-adas/test_report_demo.rst

```diff
@@ -0,0 +1,16 @@
+{% set page="test_report_demo.rst" %}
+{% include "demo_page_header.rst" with context %}
+
+.. _Test_Report_Demo:
+
+Test-Report Demo
```

Comment:
Can we name this file and the headline may be related to the needed test cases, so `sys_5_sys_quali_test_results` or similar?
We should reference it via `toctree` inside `sys_5_sys_quali_test.rst`, so that it is not part of the left toc-tree./sidebar.